src/executables/master_server_monitor.cpp

```diff
@@ -118,20 +114,18 @@ server_monitor::register_server_entry (int pid, const std::string &exec_path, co
       entry->second.set_pid (pid);
       entry->second.set_need_revive (false);
       entry->second.set_registered_time (std::chrono::steady_clock::now ());
-      fprintf (stdout,
-	       "[SERVER_REVIVE_DEBUG] : server entry has been updated in master_Server_monitor : pid : %d, exec_path : %s, args : %s\n",
-	       pid,
-	       exec_path.c_str(), args.c_str());
+      _er_log_debug (ARG_FILE_LINE,
+		     "[Server Monitor] %s : Server entry has been updated in Server monitor. (pid : %d)\n",
+		     server_name.c_str(), pid);
     }
   else
     {
       m_server_entry_map.emplace (std::move (server_name), server_entry (pid, exec_path, args,
 				  std::chrono::steady_clock::now ()));
 
-      fprintf (stdout,
-	       "[SERVER_REVIVE_DEBUG] : server entry has been registered into master_Server_monitor : pid : %d, exec_path : %s, args : %s\n",
-	       pid,
-	       exec_path.c_str(), args.c_str());
+      _er_log_debug (ARG_FILE_LINE,
+		     "[Server Monitor] %s : Server entry has been registered into Server monitor. : (pid : %d)\n",
+		     server_name.c_str(), pid);
```

Comment:
```suggestion
      _er_log_debug (ARG_FILE_LINE,
		     "[Server Monitor] Server(%s) entry has been registered. (pid : %d)\n",
		     server_name.c_str(), pid);
```

또는 함수 말미에 넣어도 될 것 같네요.

Reply:
[Server Monitor] [%s] Server entry has been updated. (pid : %d) 로 수정하였습니다.